### ISSUE TITLE
phenaki module

### ISSUE THREAD

**Issue by jusepv** [obvious-research/phenaki-cvivit#6]
Thanks for your arxing.

Can I just load the CViViT/pytorch_model.bin you posted on huggingface into CViViT with the pretrained weight?

I want to know which files should be loaded and from which part. 

Also in your code, there is only a module for cvivit and no module for phenaki.

It would be good to know how you handled the text part.

**Comment by snehasree-sony** [comment]
@jusepv  Hey, any idea about the datasets used for training C-ViViT and Phenaki?

**Comment by obvious-research** [comment]
Hello ! We did not successfully train the phenaki module. 

We gave up because Webvid is too small/noisy/low quality data for this type of training. The best we could achieve is something like this : 

![Untitled (1)](https://github.com/obvious-research/phenaki-cvivit/assets/141053078/a170722a-06b9-48ed-a91f-416f3956558b) POV Mountain biking in the swiss alps on a beautiful day

![Untitled](https://github.com/obvious-research/phenaki-cvivit/assets/141053078/16dff214-23c3-46ad-a39d-16b4d3238c6e) A fire burning


But this is not really good quality as in the paper. 

Sorry about that, I think Phenaki is not a good approach to pursue at the moment. Look into opensora, looks more interesting.
